### DISCUSSION TITLE
Compiling on 32 bit ...

### DISCUSSION THREAD

**Discussion by markov035** [JvanKatwijk/qt-dab#297] [Q&A]
To revive an atom computer with 2gb of memor, a Sony 10", I compiled qt-dab.  

Went better than I thought ... for the compile ...

But when running, I do not see the option "dabstick" under device list.  

How can that be?  

Would it be an idea somebody puts a compiled 32 bit  version qt-dab online, as tar.gz, for experimenters?

**Comment by JvanKatwijk** [comment]
Did you configure using settings in/for qmake or cmake


Op zo 13 aug 2023 om 23:15 schreef markov035 ***@***.***>:

> To revive an atom computer with 2gb of memor, a Sony 10", I compiled
> qt-dab.
>
> Went better than I thought ... for the compile ...
>
> But when running, I do not see the option "dabstick" under device list.
>
> How can that be?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JvanKatwijk/qt-dab/discussions/297>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACCPHQDYPWHVNUADDDDEWYDXVE7W3ANCNFSM6AAAAAA3O4KO5E>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Jan van Katwijk
